user_groups/user_groups.md

```diff
@@ -298,9 +299,62 @@ Response
 ```
 ---
 
-## Add or update user group permission for a given resource
+## Add user group permission for a given resource
 
-Add or update resource permission of a user group.
+Add resource permission of a user group.
+
+```apacheconfig
+POST /v1/usergroups/{userGroupId}/permissions
+```
+
+#### Path parameters
+
+| Name | Type | Description |
+| ------------- |------------- | -------------|
+| userGroupId | `string` | The user group id to request. |
+| resourceId | `string` | The resource id to add. |
```

Comment:
@WMKhant the path above does not contain the `resourceId`.

Reply:
updated